test/appium/tests/atomic/account_management/test_recover.py

```diff
@@ -62,3 +63,84 @@ def test_logcat_recovering_account(self):
+                'validation message': 'Recovery phrase is invalid'
+            },
+            {
+                'phrase': '; two three four five six seven eight nine ten eleven twelve',
+                'element to check': recover_access_view.warnings.invalid_recovery_phrase,
+                'validation message': 'Recovery phrase is invalid'
+            },
+            {
+                'phrase': phrase_outside_the_mnemonic,
+                'element to check': recover_access_view.warnings.misspelled_words,
+                'validation message': 'Some words might be misspelled'
+            },
+        ]
+
+        # we're performing the same steps changing only phrase per attempt
+        for validation in validations:
+            phrase, elm, msg = validation.get('phrase'), validation.get('element to check'), validation.get(
+                'validation message')
+            if signin_view.i_have_account_button.is_element_displayed():
+                signin_view.i_have_account_button.click()
+            recover_access_view.send_as_keyevent(phrase)
+            recover_access_view.password_input.click()
+
+            if not elm.is_element_displayed():
+                self.errors.append('"{}" message is not shown'.format(msg))
+            recover_access_view.click_system_back_button()
+
+        signin_view.i_have_account_button.click()
+        recover_access_view.send_as_keyevent(phrase_outside_the_mnemonic)
+        recover_access_view.password_input.click()
+        recover_access_view.send_as_keyevent('123456')
+        recover_access_view.sign_in_button.click()
+        recover_access_view.cancel_button.click()
+
+        if recover_access_view.cancel_button.is_element_displayed():
+            self.errors.append('Something went wrong. Probably, the confirmation pop up did not disappear')
+
+        recover_access_view.click_system_back_button()
+        signin_view.i_have_account_button.click()
+        recover_access_view.send_as_keyevent(phrase_outside_the_mnemonic)
+        recover_access_view.password_input.click()
+        recover_access_view.send_as_keyevent('123456')
+        recover_access_view.sign_in_button.click()
+        home_view = recover_access_view.confirm_phrase_button.click()
+
+        if not home_view.profile_button.is_element_displayed():
+            self.errors.append('Something went wrong. Probably, could not reach the home screen out.')
+
+        self.verify_no_errors()
+
+    @marks.testrail_id(5499)
+    def test_passphrase_whitespaces_ignored_while_recovering_access(self):
+        signin_view = SignInView(self.driver)
+        sender = transaction_senders['U']
+        passphrase = fill_string_with_char(sender['passphrase'], ' ', 3, True, True)
+        home_view = signin_view.recover_access(passphrase=passphrase)
+
+        if not home_view.profile_button.is_element_displayed():
+            self.errors.append('Something went wrong. Probably, could not reach the home screen out.')
```

Comment:
please add `self.verify_no_errors()` here

Reply:
@yevh-berdnyk please check. replaced it with simple assertion since it's the only verification here